docs/sample_config.yaml

```diff
@@ -407,6 +407,10 @@ manhole_settings:
 # sign up in a short space of time never to return after their initial
 # session.
 #
+# 'mau_appservice_trial_days' is similar to the above, but applies a different
```

Comment:
Could you also add this to the relevant section of the configuration manual (https://github.com/matrix-org/synapse/blob/develop/docs/usage/configuration/config_documentation.md#homeserver-blocking), following the format there? The idea is that hopefully we will be able to document all config options there, and get rid of any comments in the config file itself. Thanks!

Reply:
Oooh, this is cool! Absolutely